### PR TITLE
chore: hints for linguist

### PR DESCRIPTION
**Pull request by hairmare**
# Description

Fixes this in the GitHub webui by boasting with Go

image

# Issues

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# we use .gitattributes for linguist for hinting, see:
+# <https://github.com/github/linguist/blob/master/docs/overrides.md>
+
+# generated files
+README.md linguist-generated=true
+charts/*/charts/README.md linguist-generated=true
+charts/*/test//__snapshot__/*.snap linguist-generated=true
```
Comment (vmaillot) [inline .gitattributes:7]: why `//` here?

```suggestion
charts/*/test/__snapshot__/*.snap linguist-generated=true
```

**Review by tongpu — APPROVED** [review]
What @vmaillot said, but other then that: WHAT THE FUCK IS WRONG WITH YOU???

**Comment by tongpu** [comment]
One should not approve when auto-merge is enabled. :facepalm:

**Comment by hairmare** [comment]
> WHAT THE FUCK IS WRONG WITH YOU???

am addressing important issues, why?

**Pull request merged**